tests/e2e/tests/pilot/testdata/networking/v1alpha3/service-entry-tcp-scope-public.yaml

```diff
@@ -6,7 +6,7 @@ spec:
   hosts:
   - "service.tcp.com"
   addresses:
-  - "127.255.255.11"
+  - "192.168.9.251"
```

Comment:
Dont change this.. We intentionally use the 127.255.255.* IPs for various reasons (multicluster, or to ensure that this IP is not colliding with anything else on the network, and this is also tied in with the coreDNS thing we have).

Reply:
changed to 192.0.2.X